templates/shared/forms/interactive/osm-contact-us.html

```diff
@@ -0,0 +1,254 @@
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-ran-oran" name="priority-cases">
+            <label for="priority-cases-ran-oran">RAN / ORAN</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-oss-bss" name="priority-cases">
+            <label for="priority-cases-oss-bss">OSS/BSS</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-transport" name="priority-cases">
+            <label for="priority-cases-transport">Transport</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-general-it-use-case" name="priority-cases">
+            <label for="priority-cases-general-it-use-case">General IT use case</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-other" name="priority-cases" value="Other">
+            <label for="priority-cases-other">Other</label>
+          </div>
+        </div>
+      </div>
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">What is the nature of the network function? (Optional)</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="network-nature-virtual-network-function" name="network-nature">
+            <label for="network-nature-virtual-network-function">Virtual network function (VNF)</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="network-nature-cloud-native-or-kubernetes-network-function" name="network-nature">
+            <label for="network-nature-cloud-native-or-kubernetes-network-function">Cloud-native or kubernetes network function (CNF/KNF)</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="network-nature-physical-network-function" name="network-nature">
+            <label for="network-nature-physical-network-function">Physical network function (PNF)</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="network-nature-hybrid-network-function" name="network-nature">
+            <label for="network-nature-hybrid-network-function">Hybrid network function (HNF)</label>
+          </div>
+        </div>
+      </div>
+
+      <div class="pagination">
+        <a class="pagination__link--next p-button--positive" href="">Next</a>
+      </div>
+
+    </div>
+
+    <div class="js-pagination js-pagination--2 u-hide">
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">Do you want automated deployment of your network functions? (Optional)</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="automated-deployment-yes" name="automated-deployment">
```

Comment:
These yes and no options should be radio buttons not checkboxes, search for radio buttons on ubuntu.com to see an example